src/diagnostics/structs/LogpointsCollection.ts

```diff
@@ -0,0 +1,71 @@
+import * as util from 'util';
+import * as vscode from 'vscode';
+import { Logpoint } from './Logpoint';
+// tslint:disable:align
+
+export class LogpointsCollection {
```

Comment:
general concern here about keeping state in sync between your idea of logpoints & the actual logpoints that exist on the client.  Probably need some "refresh" logic that will completely refresh this data based on what the debugger claims is "truth".

Reply:
What you said is planed. We don't have a API for retrieving logpoints from server yet. After we have that (someone is working on it, so soon), we will have a "refresh" logic.